lib/zlib.js

```diff
@@ -35,7 +35,8 @@ const {
     isAnyArrayBuffer,
```

Comment:
Here we should require `deprecate` and `types` separately.

For your informations, see #26546 point 5 and 6:

![Screen Shot 2019-03-19 at 8 45 48 AM](https://user-images.githubusercontent.com/23313266/54572595-662c4980-4a23-11e9-8a14-973e3129db86.png)

Reply:
@ZYSzys I'm getting this error:
`zlib.js:34
    isAnyArrayBuffer,
    ^
TypeError: Cannot destructure property `isAnyArrayBuffer` of 'undefined' or 'null'.
    at zlib.js:33:10`

I didn't make any changes to isAnyArrayBuffer .... :/